core/server/src/main/java/alluxio/master/block/BlockMaster.java

```diff
@@ -403,15 +403,22 @@ private JournalEntry getContainerIdJournalEntry() {
    * @param tierAlias the alias of the storage tier where the worker is committing the block to
    * @param blockId the committing block id
    * @param length the length of the block
+   * @throws NoWorkerException if the workerId is not active
    */
   public void commitBlock(long workerId, long usedBytesOnTier, String tierAlias, long blockId,
-      long length) {
+      long length) throws NoWorkerException {
     LOG.debug("Commit block from workerId: {}, usedBytesOnTier: {}, blockId: {}, length: {}",
         workerId, usedBytesOnTier, blockId, length);
 
     long counter = AsyncJournalWriter.INVALID_FLUSH_COUNTER;
 
     MasterWorkerInfo worker = mWorkers.getFirstByField(mIdIndex, workerId);
+    // TODO(peis): Check lost workers as well.
```

Comment:
@peisun1115 

`getWorkerId` is sort of special, since that is done in the beginning of the workflow for a worker essentially register with the master.

I'm curious. here, after checking lost workers, what should happen?